src/open_inwoner/accounts/tests/test_auth.py

```diff
@@ -118,6 +118,29 @@ def test_registration_fails_with_invalid_last_name_characters(self):
                 }
                 self.assertEqual(response.context["form"].errors, expected_errors)
 
+    def test_registration_fails_with_non_diverse_password(self):
+        passwords = ["pass_word-123", "PASS_WORD-123", "NoDigits"]
```

Comment:
I miss a `'password123` and `'PASSWORD123'` (single case and digits)

Reply:
Both of these are rejected by the `CommonPasswordValidator`.

Reply:
You might have missed I edited the message, anyway let's add cases that fail this validator that have 

- lowercase and digits
- uppercase and digits
- only digits
- only lower
- only upper

without anything else a test reader needs to think about.

Reply:
Yeah, I missed the edit. Will incorporate the specified cases.